help_docs: Revise `mute-a-topic` help doc.

laurynmm opened the pull request:
Generally, revises a number of out of date information in the [mute-a-topic](https://zulip.com/help/mute-a-topic) help center documentation.

**Screenshot of updated help center article**:
![Screenshot from 2022-03-21 19-49-59](https://user-images.githubusercontent.com/63245456/159344256-bdaad43e-57f8-4f78-bc50-bc2c15688fdc.png)

alya commented:
I think this doesn't yet incorporate the feedback from the Paper doc?

> ​​Let’s replace the current “From the message view” instructions with instructions to use the bell-slash in the message recipient bar. The message menu option currently described is less convenient, and will soon be removed.

laurynmm commented:
> I think this doesn't yet incorporate the feedback from the Paper doc?
> 
> > ​​Let’s replace the current “From the message view” instructions with instructions to use the bell-slash in the message recipient bar. The message menu option currently described is less convenient, and will soon be removed.

Yes, I missed that comment. Will update accordingly tomorrow.

laurynmm commented:
Updated for the alternate method using the message recipient bar. Updated screenshot of changes to documentation ...

![Screenshot from 2022-03-22 11-58-01](https://user-images.githubusercontent.com/63245456/159467090-9b87f7de-cc4f-4bc0-a9d7-d94fdbb9ca13.png)

alya commented:
Cool, I think this can be merged!

I also just realized that we should add instructions for *unmuting* a topic to this page. :) That probably argues for converting the alternative options into tabs, so that we have "Mute a topic" and "Unmute a topic" as sections, with tabbed instructions underneath each one.

timabbott commented:
Merged, thanks @laurynmm!  That tabs idea sounds nice, though one concern with that approach is that I think our plan was to use tabs for the desktop/mobile client variable.

I'm not sure we really need to spell out how to unmute topics with a full description, in the sense that you can do it the exact same way you mute topics -- with the exception of the added "Muted topics" UI that we advertise.  Potentially "Browser and unmute muted topics" as a heading would help.

laurynmm commented:
I like that header change for that section, "Browse and unmute muted topics".

Since muted topics/messages don't show up in the main feed views (stream, all or recent), unmuting from the recipient bar doesn't seem like an easy thing to do - unless you've been mentioned in a message in the topic. 

And to see them in the left sidebar you have to click "more topics" for them to show up in the topic list for the stream, and then scroll/search to find it if it's not a recent topic, I feel like that's also less convenient than using the 'Muted topics' UI.
